okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java

```diff
@@ -296,6 +342,8 @@ public void start(Listener listener) {
         source = Okio.buffer(Okio.source(socket));
         sink = Okio.buffer(Okio.sink(socket));
       } catch (IOException e) {
+        // TODO(jhump): should we instead notify the listener of shutdown+terminated?
+        // (and probably do all of this work asynchronously instead of in calling thread)
```

Comment:
It's fine to remain. My comment was a bit of a +1. The shutdown+terminated isn't necessarily any better than it is doing now, but doing it async would be really nice.